content/react/concepts/lifecycle-methods/lifecycle-methods.md

```diff
@@ -186,11 +186,11 @@ class App extends React.Component {
 }
 ```
 
-**Note:** `getDerivedStateFromError()` and `componentDidCatch()` are only able to work with errors strictly related to their child components. They _are not_ compatible with errors related to event handlers.
+>**Note:** `getDerivedStateFromError()` and `componentDidCatch()` are only able to work with errors strictly related to their child components. They _are not_ compatible with errors related to event handlers.
```

Comment:
```suggestion
> **Note:** `getDerivedStateFromError()` and `componentDidCatch()` are only able to work with errors strictly related to their child components. They are not compatible with errors related to event handlers.
```

Reply:
eheh, about note still waiting for your internal discussion, about change 150 files ;)